examples/basics/runner_GUI.ipynb

```diff
@@ -0,0 +1,220 @@
+   "execution_count": 2,
+   "id": "15d478612a761c6",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-10-11T13:35:39.085137Z",
+     "start_time": "2024-10-11T13:35:39.081539Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%load_ext gradio"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c17f7d44ec3beaf6",
+   "metadata": {},
+   "source": [
+    "Next we can create the user interface using the `create_ui` function from the `UI` module. This will both create the user interface and display it in the notebook, as well as run a local server to host the user interface as a web page."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e20dd8c4301d41e0",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "## Using the Nanover Python GUI\n",
+    "\n",
+    "### Step 1: Select Simulation Type and upload files\n",
+    "Select your preferred simulation mode using the radio buttons:\n",
+    "* \"From xml\"\n",
+    "* \"From recording\"\n",
+    "\n",
+    "Prepare and upload the required simulation files:\n",
+    "* Input files\n",
+    "* Trajectory files\n",
+    "* State file\n",
+    "\n",
+    "### Step 2: Configure Simulation Parameters\n",
+    "\n",
+    "**Network Settings**\n",
+    "* Server name\n",
+    "* Port\n",
+    "\n",
+    "**Simulation Controls**\n",
+    "* Simulation FPS\n",
+    "* Frame interval\n",
+    "* Start paused\n",
+    "\n",
+    "**Physics Parameters**\n",
+    "* Include velocities\n",
+    "* Include forces\n",
+    "\n",
+    "**Output Configuration**\n",
+    "* Record trajectory\n",
+    "* Trajectory output file\n",
+    "* Shared state file\n",
+    "\n",
+    "### Step 3: Run the Simulation\n",
+    "Execute the simulation by activating the \"Run\" button. Monitor the simulation parameters in the designated output area.\n",
+    "\n",
+    "### Step 4: Stop the Simulation\n",
+    "Terminate the simulation process using the \"Stop\" button when desired."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "bb921017722d2c94",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-10-11T13:35:47.484682Z",
+     "start_time": "2024-10-11T13:35:46.947312Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "* Running on local URL:  http://127.0.0.1:7860\n",
+      "\n",
+      "To create a public link, set `share=True` in `launch()`.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div><iframe src=\"http://127.0.0.1:7860/\" width=\"100%\" height=\"500\" allow=\"autoplay; camera; microphone; clipboard-read; clipboard-write;\" frameborder=\"0\" allowfullscreen></iframe></div>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%blocks\n",
+    "demo = UI.create_ui()\n",
+    "demo"
```

Comment:
When I run this cell, the UI is still only partially visible (you have to scroll through the output of the cell, you cannot view it all at once in the notebook). Is this something you can address, i.e. make it so that the whole UI is visible in the output of the cell? It seems more difficult to use the UI when you can't see all of the parameters simultaneously